ascify art/Ascify-Art.py

```diff
@@ -329,8 +332,15 @@ def do_popup(event, frame):
 label_4 = customtkinter.CTkLabel(tabview.tab("Size"), text="Character size: 15")
 label_4.grid(row=0, column=0, sticky="wn", pady=10, padx=20)
 
-slider_size = customtkinter.CTkSlider(tabview.tab("Size"), hover=False, height=20, button_color="white",
-                                      from_=1, to=100, command=lambda e: label_4.configure(text="Character size: "+str(int(e))))
+slider_size = customtkinter.CTkSlider(
+    tabview.tab("Size"),
+    hover=False,
+    height=20,
+    button_color="white",
+    from_=1,
+    to=100,
+    command=lambda e: label_4.configure(text=f"Character size: {int(e)}"),
+)

```

Comment:
Lines `332-361` refactored with the following changes:

- Use f-string instead of string concatenation [x4]
- Remove unnecessary calls to `str()` from formatted values in f-strings [x3]